src/main/java/ch/njol/skript/expressions/ExprClamp.java

```diff
@@ -65,8 +65,8 @@ public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean is
 	protected Number[] get(Event event) {
 		Number[] numbers = values.getArray(event);
 		Double[] clampedValues = new Double[numbers.length];
-		double min = this.min.getSingle(event).doubleValue();
-		double max = this.max.getSingle(event).doubleValue();
+		double min = this.min.getSingle(event) != null ? this.min.getSingle(event).doubleValue() : 0;
+		double max = this.max.getSingle(event) != null ? this.max.getSingle(event).doubleValue() : 0;
```

Comment:
thought about possibly doing `this.min.getOptioanlSingle(event).orElse(0)`? This should work in the same way as above just seems easier to read. Suggesting this as `this.min` is not nullable only `this.min.getSingle()`